content/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md

```diff
@@ -144,15 +144,15 @@ Before adding a new SSH key to the ssh-agent to manage your keys, you should hav
 
 3. Add your SSH private key to the ssh-agent and store your passphrase in the keychain. {% data reusables.ssh.add-ssh-key-to-ssh-agent %}
    ```shell
-   $ ssh-add -K ~/.ssh/id_{% ifversion ghae %}rsa{% else %}ed25519{% endif %}
-   ```
-   {% note %}
+   $ ssh-add --apple-use-keychain ~/.ssh/id_{% ifversion ghae %}rsa{% else %}ed25519{% endif %}
+  ```
+  {% note %}
+
+  **Note:** The `--apple-use-keychain` option is in Apple's standard version of `ssh-add`, which stores the passphrase in your keychain for you when you add an SSH key to the ssh-agent. If you chose not to add a passphrase to your key, run the command without the `--apple-use-keychain` option.
 
-   **Note:** The `-K` option is Apple's standard version of `ssh-add`, which stores the passphrase in your keychain for you when you add an SSH key to the ssh-agent. If you chose not to add a passphrase to your key, run the command without the `-K` option. 
+  If you don't have Apple's standard version of `ssh-add` installed, you may receive an error. For more information on resolving this error, see "[Error: ssh-add: illegal option -- apple-use-keychain](/articles/error-ssh-add-illegal-option-k)."
```

Comment:
```suggestion
  If you don't have Apple's standard version of `ssh-add` installed, you may receive an error. For more information, see "[Error: ssh-add: illegal option -- K](/articles/error-ssh-add-illegal-option-k)."
```

We originally asked you to make this change (thanks!), but after further discussion with the team, we decided to leave the other article as is for now.